static/.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "files.exclude": {
+    "node_modules": true
```

Comment:
Currently this is superfluous for our classic editor, and useful only for our new editor.

Do we want to:

1. Improve our new editor to follow the same logic and hide `node_modules` by default?
2. OR add a `.vscode/settings.json` with this content to all starters?

My preference would be (1). This looks like an implementation detail that should not be part of our starters.

Reply:
I agree. #1